### ISSUE TITLE
Lighting issue

### ISSUE THREAD

**Issue by Processhf** [qouteall/ImmersivePortalsMod#299]
https://github.com/qouteall/ImmersivePortalsMod/issues/45

I'm having this issue on the newest 1.16 release

I'd also like help with why none of the portals will link to a portal above the nether ceiling? 
I have to manually fix every portal with commands to get it to link to our above-ceiling nether hub.

**Comment by qouteall** [comment]
This mod does not allow an overworld frame to link to the obsidian frame above nether ceiling. But you can do that by lighting up a portal on nether above ceiling.

**Comment by Processhf** [comment]
I've tried that. It will _always_ create a new portal even if the coords are exactly correct. It will place a new portal 1-2 blocks away from the portal in the overworld.

**Comment by qouteall** [comment]
What did you do to trigger the lighting issue?
About the nether portal linking issue, it can only link to the frame with the same shape and direction and there should be no block inside the frame.

**Comment by Processhf** [comment]
It would happen any time a portal was in a chunk we loaded too quickly. We
use a teleporter mod and could reproduce the lighting bug by simply
telporting next to a nether portal and the other side would have broken
lighting. Also other glitches we had included the sky flickering obscenely,
the animations of the sky and stars being in an endless choppy loop, and
overall the players lagged heavily around portals.
